website/docs/r/cdn_frontdoor_route.html.markdown

```diff
@@ -35,7 +35,6 @@ resource "azurerm_cdn_frontdoor_origin_group" "example" {
 
 resource "azurerm_cdn_frontdoor_origin" "example" {
   name                                  = "example-origin"
-  cdn_frontdoor_profile_origin_group_id = azurerm_cdn_frontdoor_origin_group.example.id
```

Comment:
This needs to be but under `cdn_frontdoor_origin_group_id` instead of `cdn_frontdoor_profile_origin_group_id`